gallery/plot_transforms.py

```diff
@@ -89,3 +89,65 @@ def _plot(img, title, **kwargs):
 # performs random affine transform on an image.
 affined_img = T.RandomAffine(degrees=(30, 70), translate=(0.1, 0.3), scale=(0.5, 0.75))(orig_img)
 plot(affined_img, "Affine transformed image")
+
+####################################
+# RandomCrop
+# ------------
+# The :class:`~torchvision.transforms.RandomCrop` transform
+# (see also :func:`~torchvision.transforms.functional.crop`)
+# performs random crop transform on an image.
+randomcrop_img = T.RandomCrop(size=(128, 128))(orig_img)
+plot(randomcrop_img, "Random Cropped transformed  image")
+
+####################################
+# RandomGrayscale
+# ------------
+# The :class:`~torchvision.transforms.RandomGrayscale`)
+# (see also :func:`~torchvision.transforms.functional.rgb_to_grayscale`)
+# performs random gray scale transform on an image.
+randomgray_img = T.RandomGrayscale(p=0.4)(orig_img)
+plot(randomgray_img, "Random Gray Scale  image")
+
+####################################
+# RandomResizedCrop
+# ------------
+# The :class:`~torchvision.transforms.RandomResizedCrop`)
+# (see also :func:`~torchvision.transforms.functional.resized_crop`)
+# performs random resize crop transform on an image.
+randresizecrop_img = T.RandomResizedCrop(size=(32, 32))(orig_img)
+plot(randresizecrop_img, "Random Resize Crop  image")
+
+####################################
+# RandomHorizontalFlip
+# ------------
```

Comment:
The doc building is failing because the underline `---` is shorter than the title. Would you mind fixing these so that every title matches the underline (here and everywhere else)